sound/soc/intel/boards/sof_ssp_amp.c

```diff
@@ -384,7 +395,8 @@ static int sof_ssp_amp_probe(struct platform_device *pdev)
 
 	mach = pdev->dev.platform_data;
 
-	dmic_be_num = mach->mach_params.dmic_num;
+	if (dmi_check_system(chromebook_platforms) || mach->mach_params.dmic_num > 0)
```

Comment:
what is the OR condition really required for? To limit to 2 DAI links only?

Reply:
There are always 2 BE DAI Links for DMICs regardless of the number of DMIC.